docs/changes/0.18.0.rst

```diff
@@ -0,0 +1,65 @@
+Changelog for QCoDeS 0.18.0
+===========================
+
+The September 2020 release of QCoDeS.
+
+
+Breaking Changes:
+_________________
+
+There are no breaking changes in this release of QCoDeS.
+
+
+New:
+____
+
+- Support nested ``Measurement`` context managers; in particular, only create
+  one background writer thread for multiple datasets (#2124)
+- Interactive widget to explore datasets in Jupyter Notebooks (#1967)
+
+
+Improved:
+_________
+
+- Jump to value when trying to set a parameter that cannot be set in steps
+  (can't be swept) (#2128)
+- Raise exception if ``get_cmd`` is not defined in a ``Group`` of
+  ``GroupParameter`` s (#2133)
+- Add new ``set_parameters`` method to ``Group`` and remove the old deprecated
+  one (#2137)
+- Update ArrayParameter Docs (#2126)
+- Add a docs page listing projects to QCoDeS project (#2142)
+
+
+Improved drivers:
+_________________
+
+- Remove warning about complex numbers in R&S ZNB driver (#2140)
+
+
+Under the hood:
+_______________
+
+- Simplify instrument destructor (#2132)
+- Add types to ``RunDescriber`` dict representations (#2134)
+- Rework ``RunDescriber`` and update to version 2 (#2135)
+- Allow ``from_dict_to_current`` to deserialize ``RunDescriber`` from new
+  versions (#2144)
+- Add ``python_requires`` as key to ``setup.py`` (#2123)
+- Fix missing comma in ``install_requires`` (#2163)
```

Comment:
I would drop this one. Its a fix to #1967 so the broken syntax has not been on master for more than 24 hours